store/src/java-test/com/zimbra/cs/service/mail/GetFolderTest.java

```diff
@@ -120,7 +120,7 @@ public void view() throws Exception {
         Assert.assertNotNull("top-level folder is listed", root);
         Assert.assertTrue("top level folder is stubbed", isStubbed(root));
 
-        Set<String> subfolderNames = ImmutableSet.of("Trash", "Briefcase", "foo", "bar", "Inbox");
+        Set<String> subfolderNames = ImmutableSet.of("Trash", "Briefcase", "foo", "bar", "Inbox", "Files shared with me");
```

Comment:
This seems to be briefcase one.

Reply:
@dasiyogesh Can you please confirm if this is okay?

Reply:
its a new system folder to list shared files, should be ok.